src/modules/common/containers/__tests__/LanguageSelector.spec.js

```diff
@@ -24,9 +24,9 @@ describe('LanguageSelector', () => {
     ar: 'test/url/ar'
   }
 
-  test('should match snapshot', () => {
-    const mockCloseDropDownCallback = jest.fn()
+  const mockCloseDropDownCallback = jest.fn()
```

Comment:
jest.fn() functions shouldn't be outside of test() environments. You probably expect a fresh mock when using it in different tests. Also if you don't expect anything from it you can just pass an empty lambda.

Reply:
done